yellowstone-grpc-geyser/src/monitor.rs

```diff
@@ -0,0 +1,52 @@
+use std::{
+    sync::{
+        atomic::{AtomicBool, Ordering},
+        Arc,
+    },
+    time::Duration,
+};
+
+use once_cell::sync::Lazy;
+use solana_client::nonblocking::rpc_client::RpcClient;
+use solana_rpc_client_api::{client_error, request};
+use tokio::time::interval;
+
+pub const HEALTH_CHECK_SLOT_DISTANCE: u64 = 100;
```

Comment:
I did not make this an env variable because of speed. I want to get this deployed asap to fix customer impact. I am a RPC land rookie and don't even know how to configure env variables.

Reply:
Done